MatchmakeSample/matchMakingAndMps.md

```diff
@@ -0,0 +1,52 @@
+# Matchmaking and Multiplayer Servers integration
+
+You can have Matchmaking integrate and allocate servers on PlayFab Multiplayer Servers, you can see the documentation [here](https://learn.microsoft.com/en-us/gaming/playfab/features/multiplayer/matchmaking/multiplayer-servers).
+
+## Preview feature - getting the matchmaking queue name after a server allocation
+
+We have recently launched a preview feature that allows you to get the matchmaking queue name after a server allocation. The way to retrieve the queue name is to search the config dictionary for the "PF_MATCH_QUEUE_NAME" string **after server is allocated**.
+
+### C#
+
+ - Make sure you grab the latest version of the C# GSDK
```

Comment:
Can the link be added here?

Reply:
wanted to do it after we would push the 1.0.0 version, but the URL defaults to the latest one, so we can use that. Thanks!